app/components/UI/QRCode.js

```diff
@@ -73,17 +73,19 @@ class ZapQRCode extends React.PureComponent {
     size: PropTypes.string,
     color: PropTypes.string,
     bg: PropTypes.string,
+    border: PropTypes.number,
     theme: PropTypes.object.isRequired
   }
 
   static defaultProps = {
     size: 'medium',
     color: 'black',
-    bg: 'white'
+    bg: 'white',
+    border: 1
   }
 
   render() {
-    let { bg, color, size, theme } = this.props
+    let { bg, color, size, border, theme } = this.props
 
     if (size === 'small') {
```

Comment:
Consider using `switch` with a `default `case to make this more robust and readable. Maybe it's a style preference though.

Reply:
refactored